demisto_sdk/commands/format/update_integration.py

```diff
@@ -235,11 +235,35 @@ def set_default_value_for_checkbox(self):
                     elif "false" == str(value).lower():
                         param["defaultvalue"] = "false"
 
+    def handle_hidden_marketplace_params(self):
+        """
+        During the upload flow, each marketplace interprets the hidden: <marketplace name> as True or False.
+        When contributing an integration from the marketplace UI, the final yml file will display hidden: False/True
+        since it comes from the marketplace bucket. When formating the file, we want it to be
+        equivalent to the master branch
```

Comment:
```suggestion
        Since it's contributed to the marketplace repo, we need the value to be identical (marketplace), rather than boolean. This function replaces booleans with marketplace values, if they exist.
```